tests/GcodeLayerThreaderTest.cpp

```diff
@@ -0,0 +1,123 @@
+//Copyright (c) 2015 Ultimaker B.V.
```

Comment:
We live in 2017. This file is copyrighted two years longer than this header would make it seem.